security/tasks.py

```diff
@@ -513,10 +514,16 @@ def get_django_command_task(command_name):
 for name in get_commands():
     if name in settings.AUTO_GENERATE_TASKS_FOR_DJANGO_COMMANDS:
         def generate_command_task(command_name):
-            @shared_task(
+            shared_task_kwargs = dict(
                 base=LoggedTask,
                 bind=True,
-                name=command_name
+                name=command_name,
+                ignore_result=True
```

Comment:
added ignore_result as default https://docs.celeryproject.org/en/stable/userguide/tasks.html#ignore-results-you-don-t-want